app/core/services/image_manipulation/image_manipulation.py

```diff
@@ -36,6 +36,7 @@ def save_image_to_buffer(
     :return: buffer pointing at the start of the file, containing raw image
```

Comment:
Update the documentation to inform that images with exif orientation tag are already handled and should not be managed in advanced